### PR TITLE
Actually escape whitespace characters instead of quoting the entire string

### PR DESCRIPTION
**Pull request by ikesyo**
This should be better than #63.

### CODE REVIEW

File: Sources/Task.swift [hunk 1]
```diff
@@ -47,12 +47,14 @@ public struct Task {
 }
 
 private extension String {
+	static let whitespaceRegularExpression = try! NSRegularExpression(pattern: "\\s")
```
Comment (NachoSoto) [inline Sources/Task.swift]: I'd make this explicitly `private`. If the visibility of the extension is changed, we still want this to remain an implementation detail.

Comment (ikesyo) [inline Sources/Task.swift]: Thanks @NachoSoto, addressed in 1e977b3.

**Comment by mdiep** [comment]
It doesn't look like we have a test for this. Would you mind adding one?

**Comment by ikesyo** [comment]
I've just added the tests. ✨

**Pull request merged**